helm/cluster-gcp/Chart.yaml

```diff
@@ -2,4 +2,4 @@ apiVersion: v2
 name: cluster-gcp
 description: A helm chart for creating Cluster API clusters with the GCP infrastructure provider (CAPG).
 type: application
-version: [[ .Version ]]
+version: "[[ .Version ]]"
```

Comment:
wasn't this supposed to be removed when using `abs`?

Reply:
But I guess I can remove it as it is not needed

Reply:
I would remove it if it's not actually required.